plugins/generator-1.21.8/neoforge-1.21.8/procedures/world_send_chat.java.ftl

```diff
@@ -1,2 +1,11 @@
-if (!world.isClientSide() && world.getServer() != null)
-	world.getServer().getPlayerList().broadcastSystemMessage(Component.literal(${input$text}), false);
\ No newline at end of file
+if (world instanceof ServerLevel _level) {
+	MutableComponent message = Component.literal(${input$text!"Message"});
+	message.withStyle(style -> style

```

Comment:
It may make sense to only call e.g. withBold if bold is enabled, so we don't do reduntant calls withBold(false) and similar for false values. Applies to all parameters and both versions. You could even do a direct call like before if all 3 are unchecked and color is #ffffff